File: app/src/main/java/com/example/khughes/machewidget/AccessTokenService.java
package com.example.khughes.machewidget;

import okhttp3.RequestBody;
import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.Headers;
import retrofit2.http.Body;
import retrofit2.http.POST;
import retrofit2.http.PUT;

public interface AccessTokenService {
    @FormUrlEncoded
    @Headers({"Content-Type: application/x-www-form-urlencoded",
            "Accept-Language: en-US",
            "Authorization: Basic ZWFpLWNsaWVudDo="})
    @POST("token")
    Call<AccessToken> getAccessToken(@Field("client_id") String client,
                                     @Field("grant_type") String grant,
                                     @Field("username") String username,
                                     @Field("password") String password
    );
}

File: app/src/main/java/com/example/khughes/machewidget/AccessTokenService.kt
package com.example.khughes.machewidget

import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST
import com.example.khughes.machewidget.AccessToken
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.Headers

interface AccessTokenService {
    @FormUrlEncoded
    @Headers(
        "Content-Type: application/x-www-form-urlencoded",
        "Accept-Language: en-US",
        "Authorization: Basic ZWFpLWNsaWVudDo="
    )
    @POST("token")
    fun getAccessToken(
        @Field("client_id") client: String?,
        @Field("grant_type") grant: String?,
        @Field("username") username: String?,
        @Field("password") password: String?
    ): Call<AccessToken?>?
}